source/Weapon.cpp

```diff
@@ -176,13 +176,23 @@ void Weapon::LoadWeapon(const DataNode &node)
 				rangeOverride = max(0., value);
 			else if(key == "velocity override")
 				velocityOverride = max(0., value);
+			else if(key == "damage dropoff")
+			{
+				hasDamageDropoff = true;
+				double maxDropoff = (child.Size() >= 3) ? child.Value(2) : 0.;
+				damageDropoffRange = make_pair(max(0., value), maxDropoff);
```

Comment:
should we also do a max(0. maxDropoff) for maxDropoff, to ensure that that one is also always positive.

Reply:
Well technically maxDropoff already can't be negative because minDropoff most be positive, and if maxDropoff is less than minDropoff then maxDropoff gets set to Range().